File: 2502-sort-the-people/2502-sort-the-people.java
class Solution {
    public String[] sortPeople(String[] names, int[] nums) {
       // String[] temp = new String[nums.length]; 
        for(int i=0;i<nums.length; i++){
            for(int j=0;j<nums.length; j++){
                if(nums[i]>nums[j]){

                    int temp= nums[i]; 
                    nums[i]= nums[j]; 
                    nums[j]= temp; 


                    String t= names[i]; 
                    names[i]=names[j]; 
                    names[j]=t; 
                }
            }
        }
        return names; 
    }
}

File: 2502-sort-the-people/2502-sort-the-people.py
class Solution(object):
    def sortPeople(self, names, heights):
        """
        :type names: List[str]
        :type heights: List[int]
        :rtype: List[str]
        """
        n= len(names)
        d= dict(zip(heights,names))
        res=[]
        sortedKeys = sorted(d.keys(),reverse=True)
        for i in sortedKeys : 
            res.append(d[i])
        return res